web/src/beta/features/Editor/useStorytelling.ts

```diff
@@ -42,6 +43,21 @@ export default function ({ sceneId }: Props) {
     [selectedStory?.pages, selectedStoryPageId],
   );
 
+  const onTimeChange = useCallback(
+    (time: Date) => {
+      return visualizerRef?.current?.timeline?.current?.commit({
+        cmd: "SET_TIME",
+        payload: {
+          start: visualizerRef?.current?.timeline?.current?.computedTimeline?.start,
+          current: time,
+          stop: visualizerRef?.current?.timeline?.current?.computedTimeline?.stop,
+        },
+        committer: { source: "storyPage", id: currentPage?.id },
+      });
+    },
+    [currentPage?.id, visualizerRef],
+  );
```

Comment:
This doesn't have anything to do with story anymore (it'll be used in field components only, right?). So please move it somewhere appropriate and make sure the commit is appropriate too (right now you are setting the committer to storyPage, which again isn't accurate)